dist/index.js

```diff
@@ -1,22 +1,34 @@
-require('./sourcemap-register.js');module.exports =
```

Comment:
do you really want to checking dist files?

Reply:
Yes, oddly enough this is the distro method! You're supposed to check in the distro with all of the dependencies bundled in. When people use a Node-based action, they are running it straight from the dist folder in the repo.